_data/toc.yaml

```diff
@@ -157,6 +155,10 @@ guides:
       title: Manage data in containers
   - path: /engine/docker-overview/
     title: Docker overview
+  - path: /release-notes/docker-ce/
```

Comment:
If I have heard of a security problem or I'm concerned that there might be a security bug that I'm exposed to when I install the very first time. As an ops person, I may have a checklist of things that must work or must not be present before installing the first time.